index.html

```diff
@@ -61,9 +60,152 @@ <h1>I’m Alexandra.   Glad   to  see      you!</h1>
+                                <li><a>CSS</a></li>
+                                <li><a>Javascript</a></li>
+                            </ul>
+                            <a class="button">See project</a>
+                        </div>
+                    </section>
+                    <section class="myproject">
+                        <div class="pheader">
+                            <img src="./img/projectfour.png">
+                            <h3 class="ptitle">Multi-Post Stories</h3>
+                        </div>
+                        <div class="p-details">
+                            <ul>
+                                <li>CANOPY</li>
+                                
+                                <li>Back End Dev</li>
+                               
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="lower_part">
+                        <ul class="project-tech-stack">
+                            <li><a>HTML</a></li>
+                            <li><a>CSS</a></li>
+                            <li><a>Javascript</a></li>
+                        </ul>
+                        <a class="button">See project</a>
+                        </div>
+                    </section>
+            </section>
+            
+            
+            <section class="about-me">
+                <h2 class="titletext">About Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and 
+                    experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
+                <p class="letconnect"> LET’S CONNECT</p>
+                           
+                       
+
+                    
+                <div class="socialmedia">
+                    <ul class="media">                           
+                        
+                        <li><a class="media"><img src="img/media-3.png"></a></li>
+                        <li><a class="media"><img src="img/media-4.png"></a></li>
+                        <li><a class="media"><img src="img/media-1.png"></a></li>
+                        <li><a class="media"><img src="img/media-5.png"></a></li>
+                        <li><a class="media"><img src="img/media-2.png"></a></li>
+                    </ul>
+
+                </div>
+
+                <a class="button">Get my resume</a>
+
+                <div class="skills">
+                    <ul>
+                        <li>Languages</li>
+                        <li> <img src="img/Langguage.png"></li>
+                        <li>Frameworks</li>
+                        <li>Skills</li>
+                    </ul>
+                </div>
```

Comment:
Great job on implementing the skills section with a list! However, there are some improvements needed to align it with the design:
- [ ] Pay close attention to the design specifications and make sure to apply the proper font styles, such as font family, font weight, and font size, to the skills list. This will help maintain consistency and visual harmony with the rest of the page.
- [ ]  Consider adjusting the spacing between the skills to match the design requirements. Ensure that there is consistent vertical spacing between each skill item to improve readability and aesthetics.
- [ ]  Align the skills list properly according to the design. Whether it's centered, left-aligned, or right-aligned, ensure that the alignment matches the design guidelines.

By incorporating these changes, you will enhance the visual presentation and consistency of the skills section, bringing it closer to the intended design. Keep up the great work!


| Your | Design|
| --- | --- |
| ![image](https://github.com/samuelkinyanjui/Portfolio-mobile-first/assets/55840999/364a3505-349f-4374-ad21-52f813bf64b0) |  ![image](https://github.com/samuelkinyanjui/Portfolio-mobile-first/assets/55840999/18782785-e19b-4d3c-b7b8-f3299c068c38) |